pkg/networkservice/common/interpose/server.go

```diff
@@ -51,6 +51,7 @@ type interposeServer struct {
 type connectionInfo struct {
 	endpointURL     *url.URL
 	interposeNSEURL *url.URL
+	requestingNSE   bool
```

Comment:
Could you provide a bug fix as a separate PR?

Reply:
OK